nemo/collections/nlp/models/nlp_model.py

```diff
@@ -394,5 +394,4 @@ def load_state_dict(self, state_dict: Mapping[str, Any], strict: bool = True):
             and "bert_model.embeddings.position_ids" in state_dict
         ):
             del state_dict["bert_model.embeddings.position_ids"]
-        results = super(NLPModel, self).load_state_dict(state_dict, strict=strict)
-        return results
+        super(NLPModel, self).load_state_dict(state_dict, strict=strict)
```

Comment:
change back to the previous code, because the function has return variables that need to be catched: https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.load_state_dict
```
results = super(NLPModel, self).load_state_dict(state_dict, strict=strict)
return results
```

Reply:
to improve readability, do sync to v1.20.0 in https://github.com/zhehuaichen/NeMo/pull/19 . This change is not done by us.